### ISSUE TITLE
Hull classifier attachment points

### ISSUE THREAD

**Issue by drNovikov** [stackcollision/GardenConquest#119]
I believe all the surfaces should be attachable, and the model itslf should look not like a beacon, but rather like a "flight recorder" or a "computer cube": 

![Airplane flight recorder](http://scrapetv.com/News/News%20Pages/Everyone%20Else/images-3/flight-recorder-inside.jpg)

![Computer cube](http://www.eweek.com/imagesvr_ce/7569/030313hpmicro01.jpg)

I understand that limiting mounting surfaces is an interesting game mechanics, but I also believe that those limits should come from something plausible and reasonable, and also functional.

For example, we cannot attach blocks to a thruser's exhaust end. We cannot attach blocks to a wheel's "road" surface. We cannot attach blocks to the business end of a cannon. The above is reasonable and comes from how things work in real life.

Hull classifiers are not functional, they don't shoot, provide thrust, or infuse oxygen. They are just for marking hulls, so restricting how players can install it does not seem right to me.

I'd consider them an advanced cryptographic device with a transponder that serves to identify a vessel. Such device could be perfectly fit into a protected box and installed in a safe compartment within a ship.

The idea of a lighthouse bulb introduced by Keen is not a good one, in my humble opinion. A seashore lighthouse bulb just does not feel right in a space game.

Of course, if the new "lighthouse" model is the one that will be incorporated into the mod, I will revert the changes of mounting points to the default beacon settings (lower half on side surfaces).

**Comment by zrisher** [comment]
> Of course, if the new "lighthouse" model is the one that will be incorporated into the mod, I will revert the changes of mounting points to the default beacon settings (lower half on side surfaces).

Yeah we're definitely using that model for now, but open to whatever changes are proposed later.

> Hull classifiers are not functional, they don't shoot, provide thrust, or infuse oxygen. They are just for marking hulls, so restricting how players can install it does not seem right to me.

What about the fact that they can remotely stop a ship and broadcast its position to their owner regardless of their distance on the map? That seems like some fairly advanced communication functionality to me.

**Comment by MrZSFG** [comment]
^^
he has a point

**Comment by drNovikov** [comment]
Okay, I will revert the changes now, and here is what I would like to propose as a hull classifier, a flight transponder:

![Flight transponder](http://www.airplanegear.com/becker/atc4401.gif)
